src/tree/images/ImageTreeItem.ts

```diff
@@ -71,32 +74,48 @@ export class ImageTreeItem extends ToolTipTreeItem {
         // In this case, an image end up with <none> tag need to be deleted using the Id.
         if (ref.endsWith('<none>')) {
             // Image is tagged <none>. Need to delete by ID.
-            ref = this._item.Id;
+            ref = this._item.id;
         }
 
-        return ext.dockerClient.removeImage(context, ref);
+        await ext.runWithDefaultShell(client =>
+            client.removeImages({ imageRefs: [ref] })
+        );
     }
 
     public async resolveTooltipInternal(actionContext: IActionContext): Promise<MarkdownString> {
         actionContext.telemetry.properties.tooltipType = 'image';
-        return resolveTooltipMarkdown(imageTooltipTemplate, { NormalizedName: this.fullTag, NormalizedSize: getCommonPropertyValue(this._item, 'Size'), ...await ext.dockerClient.inspectImage(actionContext, this.imageId) });
+
+        const imageInspection = (await ext.runWithDefaultShell(client =>
+            client.inspectImages({ imageRefs: [this.imageId] })
+        ))?.[0];
+        const associatedContainers = await ext.runWithDefaultShell(client =>
+            client.listContainers({ imageAncestors: [this.imageId] })
+        );
```

Comment:
Any reason not to wrap this in a Promise.all to allow some parallelization of the CLI calls?